CONTRIBUTING.md

```diff
@@ -129,30 +129,39 @@ This section include some tips that will help you to push source code.
 **Note:**
 > We recommend using a self-contained environment, 
 > such as [Python virtual environments](https://docs.python.org/3/tutorial/venv.html) 
-> or Anaconda environments, to cleanly separate Qiskit from other applications 
+> or [Anaconda](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#managing-python)
+> environments, to cleanly separate Qiskit from other applications 
 > and improve your experience.
 
 ### Setup with Conda virtual environment
 
+Issue the following command to create and activate the virtual environment:
+
 ``` {.sh}
 $ conda create -y -n QiskitDevenv python=3
-$ conda activate QiskitDevenv
+$ source activate QiskitDevenv
```

Comment:
why did you change `conda activate` to `source activate`? the latter is the newer way of activating the environment.

Reply:
On the documentation online, it suggested 'source activate ...'. But, it was for versions before 4.6. I will correct that.